### ISSUE TITLE
Add support for testing different CVMFS setups in the docker images for the pilot.

### ISSUE THREAD

**Issue by terjekv** [EESSI/filesystem-layer#43]
For testing CVMFS setups in the pilot, it may be a good idea for the docker images to offer three modes for setting up CVMFS:

1. Bind to stratum 0 (if that fails, we have issues)
2. Bind to a specific stratum 1 to test it, and
3. Go “live” and use Geo-location to pick the closest one.

Selecting one of these modes via an environment variable or similar would allow us to test our CVMFS setup with (relative) ease.

**Comment by bedroge** [comment]
In principle that should be possible (and I agree that it's a good idea), since you can easily override the `CVMFS_SERVER_URL` for the client (in this case: the Docker container) by making a `.local` configuration file. Only the first one may not work, since, afaik, it's quite standard to deny direct access to the Stratum 0 and only allow Stratum 1 servers to connect to it.

**Comment by terjekv** [comment]
Yeah, I'm not sure we need direct stratum 0 access, but if we did that we'd have to filter a few IP ranges that could be used. But in all honesty, if stratum 0 doesn't work for some reason, I doubt this is how we'd test it. So I supposed only options 2 and 3 are really important.
